test/fixtures/rules/std_rule_cases/L069.yml

```diff
@@ -0,0 +1,209 @@
+rule: L069
+
+test_pass_cast:
+  pass_str: |
+    select cast(1 as varchar) as bar
+    from foo;
+
+test_pass_casting_operator:
+  pass_str: |
+    select 1::varchar as bar
+    from foo;
+
+test_pass_multi_casting_operator:
+  pass_str: |
+    select 1::int::varchar as bar
+    from foo;
+
+test_pass_convert:
+  pass_str: |
+    select convert(varchar, 1) as bar
+    from foo;
+
+test_pass_3_argument_convert:
+  pass_str: |
+    select convert(varchar, 1, 126) as bar
+    from foo;
+
+test_fail_inconsistent_type_casting_prior_convert:
+  fail_str: |
+    select
+        convert(int, 1) as bar,
+        100::int::text,
+        cast(10 as text) as coo
+    from foo;
+  fix_str: |
+    select
+        convert(int, 1) as bar,
+        convert(text, convert(int, 100)),
+        convert(text, 10) as coo
+    from foo;
+
+# test_fail_inconsistent_type_casting_prior_3_argument_convert:
```

Comment:
What's this commented-out test? Is this something you're still working on?

Reply:
It is leftover which I forgot to clean up.